hack/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM golang:1.7.3
+RUN curl https://glide.sh/get | sh
```

Comment:
Is this the official way to get glide?  I am cagey about piping the results of curl to `sh` in a process that basically has root on my machine.